docs/1.0.0_project_setup.md

```
@@ -99,12 +131,14 @@ infoPlist: {
 
 ### Deployment Target
 
-Set the minimum iOS version to 15.1:
+If you are using an older React Native/Expo version, you need to set the minimum iOS version to 15.1:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)